crates/re_edit_ui/src/color.rs

```diff
@@ -0,0 +1,27 @@
+use re_types::components::Color;
+use re_viewer_context::ViewerContext;
+
+pub fn edit_color_ui(
+    _ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    value: &mut Color,
+) -> egui::Response {
+    let mut edit_color = (*value).into();
+    let response = egui::color_picker::color_edit_button_srgba(
+        ui,
+        &mut edit_color,
+        // TODO(#1611): No transparency supported right now.
+        // Once we do we probably need to be more explicit about the component semantics.
+        egui::color_picker::Alpha::Opaque,
+    );
+    *value = edit_color.into();
+
+    ui.painter().rect_stroke(
+        response.rect,
+        1.0,
+        ui.visuals().widgets.noninteractive.fg_stroke,
+    );
+
+    let [r, g, b, a] = edit_color.to_array();
+    response.on_hover_text(format!("Color #{r:02x}{g:02x}{b:02x}{a:02x}"))
```

Comment:
Might be worth adding `on_disabled_hover_text` here as well. I'm expecting that the behaviour for e.g. the "Fallback" line would be:
- hover on "Fallback" text -> tooltip explains what it is
- hover on Fallback's colour widget -> tooltip gives colour value
...which is kind of nice.

Reply:
oh I thought `on_hover_text` is triggered even when it's disabled. I have to set the same for `on_disabled_hover_text`?

Reply:
Actually I'm not entirely sure. But currently it doesn't trig for the "Fallback" item. Maybe that's another issue?